opensrp-fp/src/main/java/org/smartregister/fp/common/provider/RegisterProvider.java

```diff
@@ -66,9 +69,16 @@ public void getView(Cursor cursor, SmartRegisterClient client, RegisterViewHolde
             populatePatientColumn(pc, client, viewHolder);
             populateIdentifierColumn(pc, viewHolder);
             populateLastColumn(pc, viewHolder);
+            populateMethodExitColumn(pc, viewHolder);
         }
     }
 
+    private void populateMethodExitColumn(CommonPersonObjectClient pc, RegisterViewHolder viewHolder) {
```

Comment:
will you be needing this in future?

Reply:
Yes, this method will show the Method Exit to user on main register page, after they fill the visit form.